java/opendcs/src/main/java/decodes/cwms/validation/ScreeningAlgorithm.java

```diff
@@ -77,30 +77,25 @@ protected Screening getScreening(TimeSeriesIdentifier tsid)
 	boolean _inputIsOutput = false;
 	public boolean inputIsOutput() { return _inputIsOutput; }
 	
-	PropertySpec algoPropSpecs[] =
-	{
-		new PropertySpec("noOverwrite", PropertySpec.BOOLEAN, "(default=false) "
-			+ "Set to true to disable overwriting of output parameter."),
-		new PropertySpec("setInputFlags", PropertySpec.BOOLEAN, "(default=false) "
-			+ "Set to true to set quality flags on the input parameter."),
-		new PropertySpec("setRejectMissing", PropertySpec.BOOLEAN, "(default=false) "
-			+ "If true and the value is REJECTED, set the output flags to MISSING."),
-		new PropertySpec("noOutputOnReject", PropertySpec.BOOLEAN, "(default=false) "
-			+ "If true and the value is REJECTED, then do not write output param at all. "
-			+ "Warning: This may leave a previous value for the output param at that time slice "
-			+ "unchanged.")
-	};
 
 	@Output(type = Double.class)
 	public NamedVariable output = new NamedVariable("output", 0);
-
+	
+	
 	@org.opendcs.annotations.PropertySpec(value = "false") 
+	/** (default=false) Set to true to disable overwriting of output parameter. */
```

Comment:
```suggestion
@PropertySpec(value = "false", description="(default=false) Set to true to disable overwriting of output parameter.") 
	public boolean noOverwrite = false;
```

You can remove the prefix 'org.opendcs.annotations.'   and move comment into PropertySpec , description like above.